docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
 
   mysql:
     image: mysql:5.7
+    platform: linux/amd64
```

Comment:
The Docker desktop [documentation](https://docs.docker.com/desktop/mac/apple-silicon/#known-issues) for Apple Chip had this under known issue and this is the possible fix.